extensions/domain-analyzer/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Please us the default Raycast `extensions/bronnoysundregistrene/eslint.config.js` setup:

```
const { defineConfig } = require("eslint/config");
const raycastConfig = require("@raycast/eslint-config");

module.exports = defineConfig([
  ...raycastConfig,
]);
```